Gems/Terrain/Assets/Shaders/Terrain/Terrain.azsl

```diff
@@ -0,0 +1,126 @@
+
```

Comment:
Nitpick: Will the copyright check complain that the first line is blank instead of the copyright header?

Reply:
Nope, didn't complain, but I'll remove the blank line anyways.